### ISSUE TITLE
Missing docker tag for 1.8.2

### ISSUE THREAD

**Issue by nesl247** [ConfigHubPub/ConfigHubPlatform#101]
Seems like this is being done manually rather than via CI or Docker Hub auto builds.

**Comment by edoarsla** [comment]
Correct.  I'm not at the top of my game in this department - if you can help automate this, it would be greatly appreciated.

I've started migrating to SpringBoot/Gradle, but its still some time off.  V2, according to this plan will do everything from a single build,  but in this vanilla implementation, I could use devops help.

**Comment by nesl247** [comment]
This is actually really simple. See https://docs.docker.com/docker-hub/builds/ for more information. I'd be happy to configure it, but obviously that would require access. If you'd like I can either hop on Skype or something else to assist, or just let me know if I can assist in any way.

Thanks!
